model/src/test/kotlin/licenses/LicenseInfoResolverTest.kt

```diff
@@ -665,6 +686,7 @@ class LicenseInfoResolverTest : WordSpec() {
     private fun createLicenseInfo(
         id: Identifier,
         authors: SortedSet<String> = sortedSetOf(),
+        copyrightStatements: SortedSet<String> = sortedSetOf(),
```

Comment:
The argument should be called `copyrightHolders` also here for consistency with the called function.